modules/representation_management/spec/serializers/accredited_entities/individual_serializer_spec.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe RepresentationManagement::AccreditedEntities::IndividualSerializer, type: :serializer do
+  subject { described_class.new(individual) }
+
+  before do
+    create(:accredited_individual,
+           :with_organizations,
+           individual_type: 'representative',
+           first_name: 'Bob',
+           last_name: 'Law',
+           address_line1: '123 East Main St',
+           address_line2: 'Suite 1',
+           address_line3: 'Address Line 3',
+           address_type: 'DOMESTIC',
+           city: 'My City',
+           country_name: 'United States of America',
+           country_code_iso3: 'USA',
+           province: 'A Province',
+           international_postal_code: '12345',
+           state_code: 'ZZ',
+           zip_code: '12345',
+           zip_suffix: '6789',
+           phone: '222-222-2222',
+           email: 'email@example.com')
+  end
+
+  let(:individual) do
```

Comment:
For this serializer, you should be able to use a normal `let(:individual) {create...}` block. The `RepresentationManagement::AccreditedIndividuals::IndividualSerializer` spec had to do it through the query so that `distance` was a defined attribute on the object being serialized.

Reply:
Done!